File: C++/program77.cpp
// Input : 4 #
// Output: # # # #

// Input : 5 &
// Output: & & & & & 

// Input : 3 z
// Output: z z z

#include<iostream>
using namespace std;

void Display(int iNo, char ch)
{
    if(iNo < 0)
    {
        iNo = -iNo;
    }
    int iCnt = 0;
    
    for(iCnt = 0; iCnt < iNo; iCnt++)  
    {
        cout<<ch<<'\t';
    }
    
    cout<<"\n";

}

int main()
{
    int iValue = 0;
    char cValue = '\0';

    cout<<"Enter number "<<'\n';
    cin>>iValue;

    cout<<"Enter character"<<'\n';
    cin>>cValue;

    Display(iValue,cValue);

    return 0;
}

File: Python/program77.py
# Input : 4
# Output: 1 2 3 4 

def Display(iNo):
    if(iNo < 0):
        iNo = -iNo
        
    for i in range(1, iNo+1,1):
        print(i,"\t", end="")
    print()
    
def main():
    print("Enter number")
    iValue = int (input())
    
    Display(iValue)
    
    
if __name__ == "__main__":
    main()